src/call/client.rs

```diff
@@ -167,14 +167,14 @@ impl Call {
         mut opt: CallOption,
     ) -> Result<ClientSStreamReceiver<Resp>> {
         let call = channel.create_call(method, &opt)?;
-        let mut payload = vec![];
+        let mut payload = MessageWriter::new();
         (method.req_ser())(req, &mut payload);
         let cq_f = check_run(BatchType::Finish, |ctx, tag| unsafe {
+            let buffer = payload.byte_buffer();
```

Comment:
Probably memory leak if the call at L174 fails.

Reply:
Good catch!

Reply:
Addressed in the last commit